nss-day-cohort-48/rare-api-orchestra-outlaws
# PostReactions

#### Changes Made

1. Added `post.py` to `models` directory.
2. Added `post.py` to `views` directory
3. Modified `urls.py` to register path to '/post_reactions'
4. Modified `rebuild-database` to include recent fixtures added

#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout mec-postreactions1
   ```
2. Run rebuild-database
3. Run Postman
4. Perform GET, POST, UPDATE, and DELETE methods on /post_reactions
5. Running GET at /post_reactions{post_reaction.id} should return an individual post
6. Running GET at /post_reactions?rare_user={rare_user.id} should return all post reactions by queried rare_user
7. Running GET at /post_reactions?post={post.id} should return all post reactions by queried post
